src/serializer.ts

```diff
@@ -0,0 +1,115 @@
+import { ObjectSerializer } from "./api";
+import { V1ObjectMeta } from "./gen/model/v1ObjectMeta";
+
+export type AttributeType = {
+    name: string;
+    baseName: string;
+    type: string;
+};
+
+export class KubernetesObject {
```

Comment:
This is never used anywhere else, do we want to `export` it here for any consumer of this library to use or was this by mistake?